src/Instances/defaultProps.lua

```diff
@@ -106,5 +106,9 @@ return {
 		BackgroundColor3 = Color3.new(1, 1, 1),
 		BorderColor3 = Color3.new(0, 0, 0),
 		BorderSizePixel = 0
-	}
+	},
+
+	HandleAdornment = {},
```

Comment:
Btw, the format of `defaultProps` needs the actual class names specified individually rather than superclasses.

Reply:
Oh whoops haha completely forgot about that!